yolov3-tiny/logging.h

```diff
@@ -25,6 +25,13 @@
 #include <ostream>
 #include <sstream>
 #include <string>
+#include "macros.h"
+
+#if NV_TENSORRT_MAJOR >= 8
```

Comment:
Is this necessary since already included macros.h?

Reply:
Good point. Sorry, I copied this over from the yolov3 but you're right.

Would it make sense to remove this from yolov3 as well?